rubocop-airbnb/rubocop-airbnb.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
     'Gemfile',
   ]
 
-  spec.add_dependency('rubocop', '~> 0.75.1')
-  spec.add_dependency('rubocop-rspec', '~> 1.36.0')
-  spec.add_dependency('rubocop-performance', '~> 1.5.0')
-  spec.add_dependency('rubocop-rails', '~> 2.3.0')
-  spec.add_development_dependency('rspec', '~> 3.5')
+  spec.add_dependency('rubocop', '~> 1.39')
+  spec.add_dependency('rubocop-performance', '~> 1.15.0')
```

Comment:
fyi, there are newer versions of rubocop-performance and rubocop-rails. might want to bump those up if they are compatible. i kinda prefer >= these days, but reducing the constraints to major.minor is fine too.